openshift/template.yaml

```diff
@@ -1,81 +1,51 @@
 apiVersion: v1
 kind: Template
 labels:
-  template: bayesian-gremlin-server
+  template: oca-gremlin-server
 metadata:
-  name: bayesian-gremlin-server
+  name: oca-gremlin-server
   annotations:
-    description: bayesian-gremlin-server
+    description: gremlin server setup that creats janus graph with gremlin service and route for the same
 objects:
 - apiVersion: v1
   kind: Service
   metadata:
     labels:
-      service: bayesian-gremlin-${CHANNELIZER}${QUERY_ADMINISTRATION_REGION}
-    name: bayesian-gremlin-${CHANNELIZER}${QUERY_ADMINISTRATION_REGION}
+      type: oca-service
+      template: oca-gremlin-server-${CHANNELIZER}
+    name: oca-gremlin-${CHANNELIZER}
   spec:
     ports:
     - name: "8182"
       port: 8182
       protocol: TCP
       targetPort: 8182
     selector:
-      service: bayesian-gremlin-${CHANNELIZER}${QUERY_ADMINISTRATION_REGION}
+      service: oca-gremlin-${CHANNELIZER}
 - apiVersion: v1
   kind: DeploymentConfig
   metadata:
     labels:
-      service: bayesian-gremlin-${CHANNELIZER}${QUERY_ADMINISTRATION_REGION}
-    name: bayesian-gremlin-${CHANNELIZER}${QUERY_ADMINISTRATION_REGION}
+      type: oca-dc
+      template: oca-gremlin-server-${CHANNELIZER}
+    name: oca-gremlin-${CHANNELIZER}
   spec:
     selector:
-      service: bayesian-gremlin-${CHANNELIZER}${QUERY_ADMINISTRATION_REGION}
-    strategy:
-      type: Rolling
-      rollingParams:
-        post:
-          failurePolicy: ignore
-          execNewPod:
-            containerName: bayesian-gremlin
-            command: ["post-hook.sh"]
+      service: oca-gremlin-${CHANNELIZER}
     template:
       metadata:
         labels:
-          service: bayesian-gremlin-${CHANNELIZER}${QUERY_ADMINISTRATION_REGION}
+          service: oca-gremlin-${CHANNELIZER}
       spec:
         containers:
         - env:
           - name: REST
             value: ${REST_VALUE}
-          - name: DYNAMODB_CLIENT_ENDPOINT
-            value: ${DYNAMODB_CLIENT_ENDPOINT}
-          - name: DYNAMODB_PREFIX
-            valueFrom:
-               configMapKeyRef:
-                 name: bayesian-config
-                 key: dynamodb-prefix
-          - name: DYNAMODB_CLIENT_CREDENTIALS_CLASS_NAME
-            value: ${DYNAMODB_CLIENT_CREDENTIALS_CLASS_NAME}
-          - name: AWS_ACCESS_KEY_ID
-            valueFrom:
-              secretKeyRef:
-                name: aws-dynamodb
-                key: aws_access_key_id
-          - name: AWS_SECRET_ACCESS_KEY
-            valueFrom:
-              secretKeyRef:
-                name: aws-dynamodb
-                key: aws_secret_access_key
-          - name: AWS_DEFAULT_REGION
-            valueFrom:
-               secretKeyRef:
-                 name: aws-dynamodb
-                 key: aws_region
-          - name: JAVA_OPTIONS
```

Comment:
Don't remove JAVA_OPTIONS. It will be helpful to parameterise JVM without altering templates.

Reply:
added  back JAVA_OPTIONS